python/ray/serve/common.py

```diff
@@ -39,3 +39,34 @@ def __init__(self,
         self.deployer_job_id = deployer_job_id
         # The time when this deployment was deleted.
         self.end_time_ms = end_time_ms
+
+
+@dataclass
+class ReplicaName:
+    # Contextual input sources
```

Comment:
this comment is not helpful without context. What is a contextual input source? o.O

Reply:
ah i didn't quite optimize for readability of this part yet. I was referring to the full replica name consists of two parts: deployment tag + replica_suffix, where the name itself always carries some contextual information (I think i kinda mixed "contextual features" in ads a bit ..) that indicates which deployment you're in.